.github/workflows/periodics-daily.yaml

```diff
@@ -0,0 +1,76 @@
+name: Daily jobs
+
+on:
+  schedule:
+  - cron: "30 1 * * *"
+
+jobs:
+  periodics-mark-stale:
+    name: Mark stale issues and PRs
+    runs-on: ubuntu-latest
+    steps:
+    # Stale by default waits for 60 days before marking PR/issues as stale, and closes them after 7 days.
+    # Do not expire the first issues that would allow the community to grow.
+    - uses: actions/stale@v3.0.14
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue was automatically considered stale due to lack of activity. Please update it and/or join our slack channels to promote it, before it automatically closes (in 7 days).'
+        stale-pr-message: 'This PR was automatically considered stale due to lack of activity. Please refresh it and/or join our slack channels to highlight it, before it automatically closes (in 7 days).'
+        stale-issue-label: 'no-issue-activity'
+        stale-pr-label: 'no-pr-activity'
+        exempt-issue-labels: 'good-first-issue'
+  # This needs all the previous artifacts to be created:
+  # the image should be published, the repo should be tagged, the helm
+  # chart pushed. This ensure that the latest code merged didn't start to break from a day to another.
+  smoke-tests:
```

Comment:
this can be removed from periodics if we accept this kind of tests on PRs (which I think is best)